examples/custom-client-cxx/cli.cpp

```diff
@@ -0,0 +1,232 @@
+      std::cout << "FAILURE: Unable to get TUF metadata";
+      break;
+    case aklite::cli::StatusCode::CheckinNoMatchingTargets:
+      std::cout << "FAILURE: There is no matching target for the device";
+      break;
+    case aklite::cli::StatusCode::CheckinNoTargetContent:
+      std::cout << "FAILURE: There is no target metadata in the local path";
+      break;
+    case aklite::cli::StatusCode::TufTargetNotFound:
+      std::cout << "FAILURE: Selected target not found";
+      break;
+
+    // Possible return codes for pull and install commands
+    case aklite::cli::StatusCode::InstallationInProgress:
+      std::cout << "FAILURE: Unable to pull/install: there is an installation that needs completion";
+      break;
+    case aklite::cli::StatusCode::DownloadFailure:
+      std::cout << "FAILURE: Unable to download target";
+      break;
+    case aklite::cli::StatusCode::DownloadFailureVerificationFailed:
+      std::cout << "FAILURE: Target downloaded but verification has failed";
+      break;
+    case aklite::cli::StatusCode::DownloadFailureNoSpace:
+      std::cout << "FAILURE: There is no enough free space to download the target";
+      break;
+    case aklite::cli::StatusCode::InstallAlreadyInstalled:
+      std::cout << "FAILURE: Selected target is already installed";
+      break;
+    case aklite::cli::StatusCode::InstallDowngradeAttempt:
+      // Should not hit this error, since force_downgrade is set to true
+      std::cout << "FAILURE: Attempted to install a previous version";
+      break;
+
+    // Possible return codes for install command
+    case aklite::cli::StatusCode::InstallAppsNeedFinalization:
+      std::cout << "SUCCESS: Execute `custom-sota-client run` command to finalize installation";
+      break;
+    case aklite::cli::StatusCode::InstallNeedsRebootForBootFw:
+      std::cout << "FAILURE: Reboot is required before installing the target";
+      break;
+    case aklite::cli::StatusCode::InstallAppPullFailure:
+      std::cout << "FAILURE: Unable read target data, make sure it was pulled";
+      break;
+
+    // Possible return codes for install and run command
+    case aklite::cli::StatusCode::InstallNeedsReboot:
+      std::cout << "SUCCESS: Reboot to finalize installation";
+      break;
+    case aklite::cli::StatusCode::OkNeedsRebootForBootFw:
+      std::cout << "SUCCESS: Reboot to finalize bootloader installation";
+      break;
+    case aklite::cli::StatusCode::InstallRollbackNeedsReboot:
+      std::cout << "SUCCESS: Installation failed, rollback was successful but requires reboot to finalize";
+      break;
+
+    // Possible return codes for run command
+    case aklite::cli::StatusCode::NoPendingInstallation:
+      std::cout << "FAILURE: No pending installation to run";
+      break;
+    case aklite::cli::StatusCode::InstallRollbackOk:
+    case aklite::cli::StatusCode::InstallOfflineRollbackOk:
+      std::cout << "SUCCESS: Installation failed but rollback was successful";
```

Comment:
Not sure if it is "SUCCESS". Maybe something like "FAILURE: failed to install; SUCCESS: rollback is successful".